src/index.js

```diff
@@ -32,7 +32,7 @@ class Packager {
   }
 
   async testSymlink (comboOpts, zipPath) {
-    const testPath = path.join(this.tempBase, 'symlink-test')
+    const testPath = path.join(this.tempBase, 'symlink-test' + Math.random())
```

Comment:
```suggestion
    const testPath = path.join(this.tempBase, `symlink-test-${comboOpts.platform}-${comboOpts.arch}`)
```

Reply:
better indeed!

Reply:
Can you click "Commit suggestion" or enable "allow edits from maintainers"?